frontend/src/pages/AppDetails/AppDetailsPage.tsx

```diff
@@ -150,7 +150,12 @@ class AppDetails extends React.Component<AppDetailsProps, AppDetailsState> {
   private staticTabs() {
     const overTab = (
       <Tab title="Overview" eventKey={0} key={'Overview'}>
-        <AppInfo app={this.state.app} duration={this.props.duration} health={this.state.health} />
+        <AppInfo
+          app={this.state.app}
+          duration={this.props.duration}
+          health={this.state.health}
+          cluster={this.state.cluster ? this.state.cluster : ''}
```

Comment:
If cluster is part of app, probably the cluster parameter is not needed here?

Reply:
could be optimized, it is done similar to workloads and services, then we need to sync them as well